apps/workflows-dashboard/index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/public/favicon-16x16.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Workflow Dashboard</title>
+    <script type="text/javascript" src="/config.js"></script>
```

Comment:
the path should be relative im not sure its actually working

Reply:
This works because this will be present along with index.html in the dist
```
ls dist
assets            avatar.png        config.js         favicon-16x16.png favicon-32x32.png index.html
```